### ISSUE TITLE
ACDC Accuracy

### ISSUE THREAD

**Issue by wangkaihong** [lhoyer/HRDA#32]
Hi,

Nice work! I tried to reproduce the result on Cityscapes->ACDC and ran your code with the given config script (i.e. [uda_cityscapesHR_to_darkzurichHR_1024x1024.py](https://github.com/lhoyer/HRDA/blob/master/configs/_base_/datasets/uda_cityscapesHR_to_darkzurichHR_1024x1024.py)), but I got the test set accuracy for just 60.97% rather than the 68% reported in README. Did I miss any detail? Thanks in advance for your clarification!

**Comment by lhoyer** [comment]
Hi @wangkaihong,

Thank you for your interest in our work! You have linked the DarkZurich config, which is not suitable for ACDC. To ensure training with the correct config, I would recommend to use the automatically generated configs as defined in experiment 49: https://github.com/lhoyer/HRDA/blob/b980dece834de0b4bad2b94fc7218ae539cdaf1e/experiments.py#L538. Please note that no pseudo-label cropping is used for ACDC as it has no rectification artifacts (as you can see in experiments.py). You can run the experiment using `python run_experiments.py --exp 49`. For further instructions, please refer to the README.md.

**Comment by wangkaihong** [comment]
Hi,

Thanks for the timely response! I meant to link the ACDC one, sorry for the confusion. 

Just for clarification for the last time, plcrop is disable when ACDC is taken as the target domain dataset, while the rest of the configurations remain the same as in GTAV->Cityscapes, correct?

**Comment by lhoyer** [comment]
Yes, this should be right. You can double check it by generating the ACDC config using experiment 49 and doing a diff with the GTA->Cityscapes config.

**Comment by wangkaihong** [comment]
Great! May I check why is Cityscapes specifically chosen to use plcrop and how did you select the hyperparameters, i.e., width and height of the crops?

**Comment by lhoyer** [comment]
The Cityscapes images have rectification artifacts at the image borders (see the upper image border of the crop below for illustration). These artifacts can cause undesired pseudo-label drift. Therefore, these areas are ignored when training with pseudo-labels. The width/height of the pseudo-label crop are chosen to exclude these rectification artifacts.

![frankfurt_000000_003025_leftImg8bit_crop](https://user-images.githubusercontent.com/1277888/226301899-1f4a54c3-8126-4694-8b37-dbeb6f12b2ac.jpg)

**Comment by wangkaihong** [comment]
Thanks for the very detailed clarifications!
